### ISSUE TITLE
Missing indexes and constratins in export

### ISSUE THREAD

**Issue by kepi** [phpmyadmin/phpmyadmin#12041]
### Steps to reproduce
1. go to any table with indexes and/or foreign keys
2. click export
3. choose SQL format
4. export structure ~~data~~ only (only for convenience)

### Expected behaviour

Export should contain section with Indexes and Constraints as in version 4.5.4 or previous ones.

### Actual behaviour

Export contains only CREATE TABLE part and AUTO_INCREMENT part

### Server configuration
**Operating system**:
Debian Jessie
**Web server:**
Nginx
**Database:**
MySQL 5.6.23
**PHP version:**
7.0
**phpMyAdmin version:**
5.4.5

Same problem is on current MAINT_4_5_5 branch.

### Client configuration
**Browser:**
Google Chrome 48
**Operating system:**
Debian Sid

Export is working correctly on branch MAINT_4_5_4_1

**Comment by nijel** [comment]
I can't reproduce this. Maybe it's specific to table structure you have? Can you please post it here?

**Comment by kepi** [comment]
Yes, it looks like it is, I probably hit always "wrong" tables when testing.

I found some "parts" causing troubles while I tried to create some simple structure which will help you with reproducing this bug. Check those two files and just diff them, only difference is columns in `customers` table.

* Structure that works OK or almost OK in 4.5.5 http://pastebin.com/Syy28bNV
* Structure that doesn't work in 4.5.5 http://pastebin.com/vQYhZZa2

Here are export results (default settings) from second (not working structure) in PMA 4.5.4 vs 4.5.5
* OK on 4.5.4: http://pastebin.com/XL0JyD8n
* NOT OK on 4.5.5: http://pastebin.com/stQn4mWZ

There are more errors than I thought when creating this issue. You can also see that comments part for columns is wrong, there is " COMMENT AS..." instead of "COMMENT".

**Comment by nijel** [comment]
At least there is a bug in the SQL parser: https://github.com/phpmyadmin/sql-parser/issues/39

Maybe other issues as well, but first we need to fix this one.

**Comment by Mrodent** [comment]
I'm getting this problem.
mysql  Ver 15.1 Distrib 10.2.14-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
phpMyAdmin 4.5.4.1deb2ubuntu2
OS: Linux Mint 18.1

I have a large-ish database.  Many of tables were "inherited" from someone else's setup ... and then I added 3 more tables.  

The old table export fine, but the new tables do not export correctly: one has a primary key with auto_increment (`ID`), a non-primary key, and an FK constraint on the latter.  When exported (structure only) the ONLY instructions found in the .sql file are "CREATE TABLE... " with the fields and then "ALTER TABLE ..." to make the ID field AUTO_INCREMENT.

When attempting to copy the database within phpMyAdmin this has caused a lot of head-scratching before I managed to find the source of the problem!

This surely must qualify as a bug.

Much confusion resulted from the fact that when I do SHOW CREATE TABLE, even within phpMyAdmin, the correct CREATE TABLE command is shown (i.e. with the 2 indices and the FK constraint!).
